src/pages/Overview/Stats/TotalNominations.tsx

```diff
@@ -22,7 +22,6 @@ export const TotalNominatorsStatBox = () => {
     label: 'Total Nominators',
     stat: {
       value: totalNominators.toNumber(),
-      total: maxNominatorsCount,
```

Comment:
Can we keep it?

Reply:
We don't have this value in polkdaot API so far.